### PR TITLE
Editing page content

### PR DESCRIPTION
**Pull request by somindatommy**
## Purpose
Editing the page [1] content.

[1] - https://is.docs.wso2.com/en/5.9.0/setup/token-persistence/

### CODE REVIEW

File: en/docs/setup/token-persistence.md [hunk 1]
```diff
@@ -9,24 +9,20 @@ database (synchronous and asynchronous token persistence).
...
 
--   [Synchronous token
-    persistence](#TokenPersistence-Synchronoustokenpersistence)
--   [Asynchronous token
-    persistence](#TokenPersistence-Asynchronoustokenpersistence)
--   [Recovery flow for token
-    persistence](#TokenPersistence-Recoveryflowfortokenpersistence)
+-   [Synchronous token persistence](#synchronous-token-persistence)
+-   [Asynchronous token persistence](#asynchronous-token-persistence)
+-   [Recovery flow for token persistence](#recovery-flow-for-token-persistence)
```
Comment (YvonneW) [inline en/docs/setup/token-persistence.md]: Let's remote the TOC as there is right-hand navigation for the sub-titles

File: en/docs/setup/token-persistence.md [hunk 2]
```diff
@@ -9,24 +9,20 @@ database (synchronous and asynchronous token persistence).
...
 
-### Synchronous token persistence
+## Synchronous token persistence
 
-![](attachments/103329466/103329467.png) 
+![synchronous-token-persistence](/assets/img/103329466/103329467.png)
```
Comment (YvonneW) [inline en/docs/setup/token-persistence.md]: ../../assets/img/

File: en/docs/setup/token-persistence.md [hunk 3]
```diff
@@ -91,48 +92,39 @@ The following table describes what each of the above attributes means:
...
-    persistence](#TokenPersistence-Synchronoustokenpersistence.1)
-    -   [The flow](#TokenPersistence-Theflow.1)
-    -   [The recovery flow](#TokenPersistence-Therecoveryflow.1)
+-   [CONN\_APP\_KEY constraint](#conn_app_key-constraint)
+-   [Asynchronous token persistence](#asynchronous-token-persistence_1)
+    -   [The flow](#the-flow_1)
+    -   [The recovery flow](#the-recovery-flow_1)
+-   [Synchronous token persistence](#synchronous-token-persistence_1)
+    -   [The flow](#the-flow_1)
+    -   [The recovery flow](#the-recovery-flow_1)
```
Comment (YvonneW) [inline en/docs/setup/token-persistence.md]: Let's remove the TOC

File: en/docs/setup/token-persistence.md (hunk 1 above)
Comment (somindatommy) [reply en/docs/setup/token-persistence.md]: Fixed: 69387cab7c97dd931cffa7a97e11d3abc27ef908

File: en/docs/setup/token-persistence.md (hunk 2 above)
Comment (somindatommy) [reply en/docs/setup/token-persistence.md]: Fixed: 69387cab7c97dd931cffa7a97e11d3abc27ef908

File: en/docs/setup/token-persistence.md (hunk 3 above)
Comment (somindatommy) [reply en/docs/setup/token-persistence.md]: Fixed: 69387cab7c97dd931cffa7a97e11d3abc27ef908

**Pull request closed without merging**